src/main/resources/com/google/api/codegen/csharp/wrapper.snip

```diff
@@ -19,7 +19,7 @@
 
     // Generated code. DO NOT EDIT!
 
-    @join import : imports
+    @join import : context.removeItem(imports, context.getNamespace(service.getFile))
```

Comment:
It's to remove the "using" statement that imports the namespace within which this code is generated.
We used to do this in a different way, but the underlying framework has been altered so we can't do that any more :(